core/store/migrate/migrations/0246_cron_spec_evm_chain_id.sql

```diff
@@ -0,0 +1,5 @@
+-- +goose Up
+ALTER TABLE cron_specs ADD COLUMN evm_chain_id numeric(78,0);
```

Comment:
do we need to migrate data to popular this field? what happens if there are existing cron job specs on the node?

Reply:
It is optional, so not really. Old specs which attempted to set the field were not actually saving it, and those jobs only worked as intended on old version which include a default chain id. If we hadn't already moved past default chain id, then we could auto-populate this column, but even that would be overkill, since not all jobs need to use it.